src/main/java/EventTasks.java

```diff
@@ -0,0 +1,20 @@
+public class EventTasks extends ShowTasks {
+
+
+    ShowTasks showTasks = new ShowTasks();
+
+    public EventTasks(){
+
+    }
+
+    public void eventTask(String event){
```

Comment:
Shall we use verbs for methods/functions? The purpose of this method is not so straightforward to me. I am not sure what is this "eventTask" handling. Shall we consider to edit the name and use the name to explain?